src/Domain/Metrics/Code/Functions.php

```diff
@@ -50,6 +51,7 @@ public function getInsights(): array
             DeprecatedFunctionsSniff::class,
             NullableTypeDeclarationSniff::class,
             StaticClosureSniff::class,
+            ClosureToArrowFunctionRector::class,
```

Comment:
It's the only Rector you want to add ? Is there is other useful Rectors that we should add in default configuration ?

Reply:
@Jibbarth Thanks a lot :heart: Yes you should test it out. At this point I really need to make sure it is fully functional.

Reply:
No it's just for demonstration and test. I will prepare a list of Rectors and then we can consider to add them to default config.